### DISCUSSION TITLE
Can we support "HighGo Database"?

### DISCUSSION THREAD

**Discussion by UESTC-zzy** [go-pg/pg#2009] [Ideas]
In the project, it is found that it is impossible for the go-pg to be connected to the "HighGo Database". This is mainly because the go-pg cannot support the sm3 authentication algorithm, so the link cannot be established and the subsequent ddl operations cannot be performed.Can we consider adding support for sm3 certification for connecting to "HighGo Database"?

**Comment by elliotcourant** [comment]
If you have some clear documentation on the exact protocol used by the database then I can take a look. But overall its unlikely to be added only because this project is in maintenance mode only at the moment and not looking to add new features.

**Reply by UESTC-zzy** [reply]
It seems that it is not reasonable for you to add a specific logic for the "HighGo Database". Let me think about the specific solution.

**Reply by bruceauyeung** [reply]
> It seems that it is not reasonable for you to add a specific logic for the "HighGo Database". Let me think about the specific solution.

what's your solution in the end?
